app/digests/tests/test_digest_api.py

```diff
@@ -41,30 +43,33 @@ def test_has_expected_data_in_response(client: Client,
 
 
 @pytest.mark.django_db
-def test_can_get_digest_by_id(client: Client,
-                              digest: Digest,
+def test_can_get_digest_by_id(auth_header: Dict,
+                              client: Client,
+                              preview_digest: Digest,
```

Comment:
`preview_digest` is a nice way to show the state of the digest when requesting the fixture